webodf/lib/ops/OpRemoveAnnotation.js

```diff
@@ -92,8 +92,8 @@ ops.OpRemoveAnnotation = function OpRemoveAnnotation() {
         // The specified position is the first walkable step in the annotation. The position is always just before the first point of change
         odtDocument.emit(ops.OdtDocument.signalStepsRemoved, {position: position > 0 ? position - 1 : position});
 
+        odtDocument.getOdfCanvas().rerenderAnnotations();
         odtDocument.fixCursorPositions();
-        odtDocument.getOdfCanvas().refreshAnnotations();
```

Comment:
Yes, the one in `OdtDocument.cloneDocumentElement`, the very trigger for the need of the #830 fix.